client.go

```diff
@@ -222,12 +222,20 @@ func (r *marathonClient) apiDelete(uri string, post, result interface{}) error {
 }
 
 func (r *marathonClient) apiCall(method, uri string, body, result interface{}) error {
+
 	// Get a member from the cluster
 	marathon, err := r.cluster.GetMember()
 	if err != nil {
 		return err
 	}
-	url := fmt.Sprintf("%s/%s", marathon, uri)
+
+	var url string
+
+	if r.config.DCOSToken != "" {
```

Comment:
Perhaps in another PR we can change this logic and place into Config itself ... i.e 
```GO
marathonURL := "http://10.241.1.71:8080/marathon,http://10.241.1.81:8080/marathon"
config := marathon.NewDefaultConfig()
config.URL = marathonURL
```
I think at the moment, the GetMember() will only return scheme://hostname ... I'll chunk it into an issue